1_fetch/src/get_sb_data.R

```diff
@@ -0,0 +1,170 @@
+getbabies <- function(idin) 
```

Comment:
Simple formatting edit - we are using snake_case (I think that is what it is called) so lets change this from 'getbabies' to 'get_babies'. Same for other function and variable names throughout.